src/main/openapi/ds-datahandler-openapi_v1.yaml

```diff
@@ -330,7 +330,7 @@ components:
           example: 'H.C. Andersen'
         filter:
           type: string
-          enum:  ['direct', 'preservica']
+          enum:  ['direct', 'preservica', 'preservica5']
```

Comment:
Please add a description that explain what `preservica` is, compared to `preservica5`.

Reply:
Very nice now!